README.md

```diff
@@ -56,6 +56,18 @@ Using Gosling.js, users can easily create interactive and scalable genomic visua
 </table>
 </div>
 
+## Why Gosling
+
+The Gosling's key features compared to existing visualization libraries and grammars are as follows:
+
+- **Data Scalability**: As Gosling is built on [HiGlass](http://higlass.io/), the grammar allows you to handle and visualize large genome-mapped data and quickly switch the scale of visualization from base-pair resolution to whole genome.
+
+- **Encoding Scalability**: Gosling supports an advanced zooming technique, called [*Semantic Zooming*](https://infovis-wiki.net/wiki/Semantic_Zoom), which allows flexible and seamless visual exploration of large genome-mapped data. This allows you to dynamically switch between different visual encoding strategies that are appropriate for given zoom scale. <!--For example, you can show nucleotide bases of genomic sequence when zoomed in while show overall distribution of the bases using stacked bar chart when zoomed out.-->
+
+- **Expressiveness**: Gosling is designed based on the [taxonomy for genomic data visualization](https://onlinelibrary.wiley.com/doi/full/10.1111/cgf.13727). Gosling supports genomic visualizations that cannot be easily created uisng general visaulization tools. For example, Gosling allows you to use a circular layout (i.e., displaying genomic coordinates in a polar system), in addition to a linear layout, and easily switch between the two.
```

Comment:
I am worried that if we say most part, people might wonder why not cover all parts of the taxonomy

Reply:
Good point, I like the edited descriptions.